query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -74,10 +75,12 @@ final case class AggregateMapReduce(aggrOp: AggregationOperator,
       sourceSchema.fixedVectorLen.filter(_ <= querySession.queryConfig.fastReduceMaxWindows).map { numWindows =>
         RangeVectorAggregator.fastReduce(aggregator, false, source, numWindows)
       }.getOrElse {
-        RangeVectorAggregator.mapReduce(aggregator, skipMapPhase = false, source, grouping)
+        RangeVectorAggregator.mapReduce(aggregator, skipMapPhase = false, source, grouping,
```

Comment:
Can you pls confirm that this code runs during the monix query pipeline setup stage, and not during scan stage?

Reply:
@vishramachandran yes this is before scanning.